pages/common/vmtouch.md

```diff
@@ -0,0 +1,24 @@
+# vmtouch
+
+> Manage filesystem cache.
+> More information: <https://manned.org/vmtouch>.
+
+- Print cache status of a file:
+
+`vmtouch {{path/to/file}}`
+
+- Load a file into cache:
+
+`vmtouch -t {{path/to/file}}`
+
+- Evict a file from cache:
+
+`vmtouch -e {{path/to/file}}`
+
+- Lock a file in memory to prevent eviction:
```

Comment:
```suggestion
- Lock a file in memory to prevent removal:
```

Reply:
Why? Eviction is correct. Or would more verbose "...prevent eviction from memory" be better?

Reply:
It is, but not everyone knows what this word mean.

Reply:
Removal would refer to removal from disk. I'll just make it more verbose to give more context.